### DISCUSSION TITLE
Equivalent solution for papers

### DISCUSSION THREAD

**Discussion by vasnt** [snipe/snipe-it#13275] [Q&A]
I ocassionally struggle to find documents. is there any tool available that can help to manage/relocate hard-copy documents/papers?

**Comment by TShiremanLCOG** [comment]
This is outside the scope of SnipeIT. Your best bet is to look into Digital Asset Management tools or a full featured Document Management System. Within a DMS you can likely add a 'document location' field to note a specific filing cabinet or other location.
